metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -198,6 +199,7 @@ class MetalsLanguageServer(
   private var renameProvider: RenameProvider = _
   private var documentHighlightProvider: DocumentHighlightProvider = _
   private var formattingProvider: FormattingProvider = _
+  private var syntheticsDecorator: SyntheticDecorationProvider = _
```

Comment:
NIT: Can we align with plural or not?

```suggestion
  private var syntheticDecorator: SyntheticDecorationProvider = _
```
or 

```suggestion
  private var syntheticsDecorator: SyntheticsDecorationProvider = _
```

Reply:
Fixed to display plural in both.